docs/content/latest/develop/learn/ttl-data-expiration-ycql.md

```diff
@@ -108,6 +108,51 @@ SELECT * FROM pageviews;
+
+##### --sst_files_soft_limit = \[number of expected files at steady state + 20\]
+##### --sst_files_hard_limit = \[number of expected files at steady state + 40\]
+
+The value of these flags is directly dependent on the number of files expected to hold the full dataset at steady state. These flags throttle writes to YCQL if the number of files per tablet exceed their value. Thus, they need to be set in a way that will In the example above, 30 files should hold 90 days worth of data. In this case, *sst_files_soft_limit* would be set to 50, and *sst_files_hard_limit* set to 70.
+
+In some fresh dataset cases, new data will be backfilled into the database before the application is turned on. This backfilled data may have a value-level TTL associated with it that is significantly lower than the `default_time_to_live` property on the table, with the desired effect being that this data be removed earlier than the table TTL would allow. By default, such data will *not* expire early. However, the **file_expiration_value_ttl_overrides_table_ttl** flag can be used to ignore table TTL and expire solely based on value TTL.
+
+**NOTE:** When using the `file_expiration_value_ttl_overrides_table_ttl` flag, be sure to set the flag back to `false` *before* all backfilled data has fully expired. Failing to do so can result in unexpected loss of data. For example, if the `default_time_to_live` is 90 days, and data has been backfilled with value-level TTL from 1 day to 89 days, it is important that the `file_expiration_value_ttl_overrides_table_ttl` flag be set back to `false` within 89 days of data ingestion to avoid data loss.
+
+#### Configuring for existing YCQL time series datasets with TTL
+
+To convert existing YCQL tables to ones configured for file expiration, the same TServer flag values as above can be used. However, it should be known that a **temporary 2x space amplification** of the data should be expected in this case. This amplification happens because the existing file structure will have kept most data in a single large file, and that file will now be excluded from compactions going forward. Thus, this file will be unchanged until its contents has entirely expired, roughly *TTL amount of time* after the file expiration feature was configured.
+
+Additionally, if data files were created with YugabyteDB versions below 2.6.6 or 2.8.1, files may lack the necessary metadata to be expired naturally. The **file_expiration_ignore_value_ttl** flag can be set to `true` to ignore the missing metadata. 
+
+**NOTE:** To prevent early data deletion, it is very important that in these cases, the `default_time_to_live` for any tables with TTL should be set to greater than or equal to the largest value-level TTL contained within those tables. It is recommended that once the files lacking the metadata have been removed, the `file_expiration_ignore_value_ttl` flag be set back to `false` (no restart required).
```

Comment:
{{< warning title="Warning" >}}
To prevent early data deletion, it is very important that in these cases, the `default_time_to_live` for any tables with TTL should be set to greater than or equal to the largest value-level TTL contained within those tables. It is recommended that once the files lacking the metadata have been removed, the `file_expiration_ignore_value_ttl` flag be set back to `false` (no restart required).
{{< /warning >}}